pkg/conversion/convert.go

```diff
@@ -21,6 +21,12 @@ type analyzerRules struct {
 }
 
 func ConvertWindupRulesetsToAnalyzer(windups []windup.Ruleset, baseLocation, outputDir string) (map[string]*analyzerRules, error) {
+	// Write discovery rules
+	err := convertDiscoveryRules(outputDir)
```

Comment:
not a blocker, but `writeDiscoveryRules` may be more correct since we're not really converting anything